extensions/spring/spring-boot-autoconfigure/src/main/java/org/axonframework/extension/springboot/autoconfig/EventProcessingAutoConfiguration.java

```diff
@@ -34,34 +35,29 @@
  * @since 4.0
  */
 @AutoConfiguration
-@EnableConfigurationProperties(EventProcessorProperties.class)
+@AutoConfigureAfter(name = {
+        // Ensure Boot’s config props infra is in place
+        "org.springframework.boot.autoconfigure.context.ConfigurationPropertiesAutoConfiguration",
+        // In Servlet mode, ensure Tomcat + MVC complete first (no CNFE in WebFlux because name-based)
+        "org.springframework.boot.autoconfigure.web.servlet.ServletWebServerFactoryAutoConfiguration",
+        "org.springframework.boot.autoconfigure.web.servlet.WebMvcAutoConfiguration"
```

Comment:
How are we ensured any other auto configuration classes in the world aren't causing similar issues? 

If we can only make a webmvc-specific solution, so be it, but if there's a generic solution to be found, I'd much prefer that.